src/robots/robot-wrapper.cpp

```diff
@@ -249,14 +249,15 @@ namespace tsid
                                           const Model::FrameIndex index,
                                           Data::Matrix6x & J) const
     {
-      return se3::getFrameJacobian<false>(m_model, data, index, J);
+      return se3::getFrameJacobian(m_model, data, index, J);
```

Comment:
getFrameJacobian only returns the Jacobian of the Frame expressed in the local Frame coordinate system
getFrameJacobian<se3::WORLD> would be equivalent to getJacobian<se3::WORLD> called with frame.parent as joint index information.